.run/desktop/test.run.xml

```diff
@@ -10,7 +10,7 @@
       </option>
       <option name="taskNames">
         <list>
-          <option value=":mpp:testDesktop" />
+          <option value="desktopTest" />
```

Comment:
Will it run all desktopTest? May be we can remove desktopCICheck then and just run desktopTest on CI?

Reply:
forgot to mention that we add API checks [here](https://github.com/JetBrains/compose-multiplatform-core/pull/803)

Reply:
So, you want later on use desktopCICheck config for API checks also?

Reply:
Yes, please, [review it](https://github.com/JetBrains/compose-multiplatform-core/pull/803) as well :)